script.js

```diff
@@ -17,4 +17,130 @@ document.querySelectorAll('.nav-list').forEach((e) => e.addEventListener('click'
+    technologies: ['html', 'Ruby on rails', 'css'],
+    technologies2: ['css', 'html', 'Bootstrap', 'Ruby'],
+    liveVersion: '#',
+    source: '#',
+  },
+  {
+    id: 4,
+    name: 'Multi - Post stories4',
+    featuredImage: 'placeholder.png',
+    featuredImage2: 'modal.png',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    desc:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    technologies2: ['css', 'html', 'Bootstrap', 'Ruby'],
+    liveVersion: '#',
+    source: '#',
+  },
+];
+
+const mainBody = document.querySelector('.main-body');
+const windowPop = document.querySelector('.window-popup');
+const wrapper = document.querySelector('.wrapper');
+
+const mainContainer = projects.map((project) => `<div
+class="projects-container hide-in-desktop show">
+<div
+    class="image-placeholder hide-in-desktop show">
+    <img src="IMG1/${project.featuredImage}"
+        alt="${project.name}">
+</div>
+<div
+    class="content hide-in-desktop show">
+    <div class="text-area">
+        <h2>${project.name}</h2>
+        <p>${project.description}</p>
+    </div>
+    <ul class="list-tags">
+        ${project.technologies2.map((technology) => `<li class="border-right">${technology}</li>`).join('')}
+    </ul>
+    <button type="button" onclick="showModal(${project.id})"
+        class="myprojects">See
+        projects</button>
+</div>
+</div>`).join('');
+
+// append the main-container to the main-body container
+mainBody.innerHTML += mainContainer;
+
+// create the modal pop up
+
+const windowContent = (modal) => {
+  const template = document.createElement('template');
+  template.innerHTML = ` <div class="heading">
+<h2>${modal.name}</h2>
+<button class="close" onclick="closeModal()" >&times</button>
+</div>
+<img src="IMG1/${modal.featuredImage2}" alt="${modal.name}">
+<p>${modal.desc}</p>
+<ul class="tags">
+${modal.technologies.map((technology) => `<li>${technology}</li>`).join('')}
+</ul>
+<div class="click-me">
+<button type="button" class="see-live">See live</button>
+<button type="button" class="source">See source</button>
+</div>`;
+  windowPop.appendChild(template.content);
+  wrapper.classList.add('active');
+};
+
+const showModal = (id) => {
+  projects.find((item) => {
+    if (Number(id) === Number(item.id)) {
+      windowContent(item);
+    }
+    return undefined;
+  });
+};
```

Comment:
- [ ] Great job rendering your project dynamically from JavaScript 👍 Your popup window needs some additional styling before it is good to go. Below is a preview of how your popup responds in different screen sizes. Notice that the styling completely breaks down between `580px` and `992px`. Then from `992px` to `1200px`, the text overlaps. Kindly fix this issue so as to follow the template.
- [ ]  Also ensure that the information in the popup window is always centralized as in the template 👍 .
- [ ] Also your `x` icon should look like that in the template. So it will be good to get rid of the extra styles on the `x` icon to close the popup 💯 
- Your site:
![Animation](https://user-images.githubusercontent.com/85113567/158029204-80e4bcd4-0906-47b6-9766-e620ab940065.gif)